### PR TITLE
Use InAppUpdatePriority for In app updates

### PR DESCRIPTION
**Pull request by JaerenMoneyBox**
**Description**

This change allows you to specify a priority using the [InAppUpdatePriority](https://developer.android.com/reference/com/google/android/play/core/appupdate/AppUpdateInfo#updatePriority()) in [Edits.tracks.releases](https://developers.google.com/android-publisher/api-ref/edits/tracks).

This can be used to set the priority of a particular release to define whether this release should be suggested/required to be upgraded to using [in app updates](https://developer.android.com/guide/playcore/in-app-updates).

This number can be between 0 and 5 (0 being lowest, 5 being highest priority).  If this value is not specified then it is set to 0.  See more info about priority [here](https://developer.android.com/guide/playcore/in-app-updates#check-priority).

**Code changes**

This change contains the following code changes:
- Updating deps (specifically google.golang.org/api to 0.21.0 that contains InAppUpdatePriority)
- Adding new input variable "UpdatePriority" that must be between 0 and 5, but is not required.
- Use this variable when creating a track release

### CODE REVIEW

File: publish.go [hunk 1]
```diff
@@ -199,6 +199,10 @@ func createTrackRelease(whatsNewsDir string, versionCodes googleapi.Int64s, user
 		newRelease.UserFraction = userFraction
 	}
 
+	if updatePriority != 0 {
```
Comment (Szabadember) [inline publish.go:202]: Is this check needed?

File: config.go [hunk 2]
```diff
@@ -19,6 +19,7 @@ type Configs struct {
 	ExpansionfilePath string          `env:"expansionfile_path"`
 	Track             string          `env:"track,required"`
 	UserFraction      float64         `env:"user_fraction,range]0.0..1.0["`
+	UpdatePriority    int             `env:"update_priority,range]0..5["`
```
Comment (Szabadember) [inline config.go:22]: According to the docs, the range should be inclusive-inclusive instead of exclusive-exclusive

**Comment by Szabadember** [comment]
Dear @JaerenMoneyBox ,
thank you for your contribution. I moved your changes to separate PRs to be able to fix the conflicts and to apply the minor changes I suggested. See PRs #100 and #101 . These changes will be released with 3.1.2

**Comment by JaerenMoneyBox** [comment]
Thanks very much for merging this @Szabadember!

**Pull request closed without merging**